app/controllers/api/tasks_controller.rb

```diff
@@ -10,15 +10,14 @@ def show
   def create
     params.require(:client_id)
     params.require(:description)
+    params.require(:user_id)
```

Comment:
I think you can create a method that returns `params.require(:task).permit(...)` then you can just do `Task.new(task_params)`.

Reply:
+1